string Hostname
float64 CPU_Count
float64 CPU_usage_perct
float64 Mem_usage_perct
float64 total_mem
float64 mbs_used
float64 uptime
float64 load1
float64 load5
float64 load15

